apps/typeform/lambda/app.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
This can be removed.
```suggestion

```

Reply:
Will keep it as it is since we are using plain JS / Node. Also the linter will cry